tests/test_jira.py

```diff
@@ -44,6 +44,7 @@ class TestJiraIssue(AbstractServiceTest, ServiceTest):
             'issuetype': {'name': 'Epic'},
             'status': {'name': 'Open'},
             'subtasks': [{'key': 'DONUT-%s' % subtask} for subtask in arbitrary_subtask_ids]
```

Comment:
Missing a comma at the end of this line.

Reply:
do'h 🤦 I'll see if I can get the test suite running locally before pushing again